src/quickgui/qgsquickpositionkit.cpp

```diff
@@ -172,11 +201,12 @@ void QgsQuickPositionKit::onSimulatePositionLongLatRadChanged( QVector<double> s
   }
 }
 
-double QgsQuickPositionKit::accuracyIndicatorWidth( QgsQuickMapSettings *mapSettings )
+// TODO fix @vsklencar - always scpm == 0
+double QgsQuickPositionKit::screenAccuracy()
 {
   if ( accuracy() > 0 )
   {
-    double scpm = QgsQuickUtils().screenUnitsToMeters( mapSettings, 1 ); // scpm is how much meters is 1 pixel
+    double scpm = QgsQuickUtils::screenUnitsToMeters( mapSettings(), 1 ); // scpm is how much meters is 1 pixel
```

Comment:
what if mapsettings() is nullptr? I think at the beginning of the function you need to check if map settings is nullptr and if yes, return 0 or so. Also in all other function that uses map settings. See scalebar implemetnation